src/main/java/Duke.java

```diff
@@ -1,10 +1,362 @@
+                    addToList(t, Tasks, Tasks.size());
+                    printTask(t, Tasks.size());
+                } else if (str.contains("deadline")) {
+                    checkDeadlineString(str);
+                    checkFormat(str);
+                    String time = str.substring(str.indexOf("/") + 3, str.length());
+                    if (str.length() - 9 + 11 > maxlength)
+                        maxlength = str.length() - 9 + 11;//- length of "deadline " + "1. [X][X]" + "(xx:" + ")"
+                    Deadline t = new Deadline(str.substring(9, str.indexOf("/") - 1), time);
+                    addToList(t, Tasks, Tasks.size());
+                    printTask(t, Tasks.size());
+                } else if (str.contains("event")) {
+                    checkEventString(str);
+                    checkFormat(str);
+                    String time = str.substring(str.indexOf("/") + 3, str.length());
+                    if (str.length() - 6 + 11 > maxlength) maxlength = str.length() - 6 + 11;
+                    Event t = new Event(str.substring(6, str.indexOf("/") - 1), time);
+                    addToList(t, Tasks, Tasks.size());
+                    printTask(t, Tasks.size());
+                } else if (str.contains("delete")) {
+                    String numberOnly = str.replaceAll("[^0-9]", "");
+                    if(numberOnly.length() == 0) numberOnly = "0"; // to avoid empty string error
+                    int num = Integer.parseInt(numberOnly);
+                    checkNum(num, Tasks.size());
+                    Task t = Tasks.get(num - 1);
+                    Tasks.remove(num - 1);
+                    printDelete(t, Tasks.size());
+                } else if (str.contains("done")) {
+                    String numberOnly = str.replaceAll("[^0-9]", "");
+                    if(numberOnly.length() == 0) numberOnly = "0"; // to avoid empty string error
+                    int num = Integer.parseInt(numberOnly);
+                    checkNum(num, Tasks.size());
+                    Tasks.get(num - 1).setDone(true);
+                    printDone(Tasks.get(num - 1));
+                } else {
+                    FormatError();
+                }
+            } catch (AssignException e){
+                  printString("OOPS! not assigned");
+            } catch (DeadlineStringException e){
+                  printString("OOPS! the deadline description cannot be empty");
+            } catch (EventStringException e) {
+                  printString("OOPS! the event description cannot be empty");
+            } catch (FormatException e) {
+                  printString("OOPS! what's the time?");
+            } catch (TodoStringException e) {
+                  printString("OOPS! the todo description cannot be empty");
+            } catch (TypingException e) {
+                  printString("OOPS! what do u mean?");
+            } catch (EmptyListException e) {
+                  printString("OOPS! empty list");
```

Comment:
Try to add more enter key lines with this large else if statement for legibility.

Also, consider changing else if statements to a switch case statement.